test/features/organization_membership_test.rb

```diff
@@ -0,0 +1,40 @@
+require "test_helper"
+
+class EditOrganizationTest < Capybara::Rails::TestCase
```

Comment:
This test appears to be testing whether a user can join an organization. Might rename the `JoinOrganizationTest`, since `EditOrganizationTest` is about editing its attributes.